Combobox.js

```diff
@@ -930,6 +945,9 @@ define([
 				var promise = sup.apply(this, arguments);
 
 				return promise.then(function () {
+					this.setAttribute("aria-owns", this.dropDown.id);
+					this.popupStateNode.setAttribute("aria-controls", this.dropDown.id);
```

Comment:
We talked about making `popupStateNode` point to the root node and then all you need is to set `aria-controls` on the `<input>`.